indico/modules/events/registration/client/js/form_submission/RegistrationFormSubmission.jsx

```diff
@@ -85,13 +91,14 @@ ConsentToPublish.propTypes = {
 
 export default function RegistrationFormSubmission() {
   const sections = useSelector(getNestedSections);
-  const {submitUrl, registrationData, initialValues} = useSelector(getStaticData);
+  const {submitUrl, registrationData, initialValues, captchaRequired} = useSelector(getStaticData);
   const isUpdateMode = useSelector(getUpdateMode);
   const isModerated = useSelector(getModeration);
   const isManagement = useSelector(getManagement);
   const publishToParticipants = useSelector(getPublishToParticipants);
   const publishToPublic = useSelector(getPublishToPublic);
   const showConsentToPublish = !isManagement && publishToParticipants !== 'hide_all';
+  const hasCaptchaPlugin = getPluginObjects('regformCaptcha').length > 0;
```

Comment:
You could do `const pluginCaptcha = renderPluginComponents('regformCaptcha');` and use that below. In the end it just returns a list as well so you can use that to check the length.